src/Controls/src/Core/BindableProperty.cs

```diff
@@ -45,7 +45,11 @@ public sealed class BindableProperty
 			{ typeof(Uri), new UriTypeConverter() },
 			{ typeof(Easing), new Maui.Converters.EasingTypeConverter() },
 			{ typeof(Maui.Graphics.Color), new ColorTypeConverter() },
-			{ typeof(ImageSource), new ImageSourceConverter() }
+			{ typeof(ImageSource), new ImageSourceConverter() },
+#if NET6_0_OR_GREATER
+			{ typeof(DateTime?), new DateTimeTypeConverter() },
```

Comment:
this is where we declare some built-in conversions for bindings. automatically convert to from DateTime-DateOnly make sense.

is the nullability marker required ?

Reply:
Yes, the nullability marker is required because the TypeConverter registration must exactly match the property types. Since DatePicker.Date is DateTime? and TimePicker.Time is TimeSpan?, the converters must be registered using the nullable types.

Reply:
so it's the case for other types as well, I should look at that

Reply:
The current implementation supports both nullable and non-nullable values, so no separate binding conversion is needed.